packages/base/src/locale/getEffectiveDir.ts

```diff
@@ -1,28 +1,9 @@
-import { getRTL } from "../config/RTL.js";
-
-const GLOBAL_DIR_CSS_VAR = "--_ui5_dir";
```

Comment:
The var is defined in a selector in `SystemCSSVars.css`, you can remove it from there as well.

Reply:
yes, also the "config/RTL.js" - I wanted to get common agreement  - I will remove everything that becomes obsolete.